src/mbgl/map/tile_worker.cpp

```diff
@@ -287,6 +287,6 @@ void TileWorker::createSymbolBucket(const GeometryTileLayer& layer,
 
 void TileWorker::insertBucket(const std::string& name, std::unique_ptr<Bucket> bucket) {
     if (bucket->hasData()) {
-        result.buckets.emplace_back(name, std::move(bucket));
+        result.buckets.emplace(name, std::move(bucket));
```

Comment:
I don't think so -- if we are, there's a bug in the bucket de-duping step. The name is the bucket name, which should be unique.